__tests__/optimize-imports.spec.ts

```diff
@@ -3,19 +3,15 @@ import * as config from '@ic/config';
 import fs from 'fs';
 import { Config } from '@ic/types';
 import { readmeExample, comments, testCase } from './optimize-imports-mocks';
+import { defaultConfig } from '@ic/defaultConfig';
```

Comment:
Detail, but this import should come before the `'./optimize-imports-mocks'` import

Reply:
Why should it come from there? it's the actual default config, it should come from the conductor since it's not a mock.

Reply:
The place you import it from is correct. I mean that the imports should be reordered. Something like this:

````typescript
import fs from 'fs';
import { Config } from '@ic/types';
import { defaultConfig } from '@ic/defaultConfig';

import { readmeExample, comments, testCase } from './optimize-imports-mocks';
````

Reply:
So basically what import-conductor does ;)

Reply:
Ohh you meant that the import order is incorrect? weird since it should run on the staged files, I'll run it manually 👍